src/store/messagesStore.js

```diff
@@ -1351,12 +1354,15 @@ const actions = {
 			})
 			// The response return an array with the reaction details for this message
 			const response = await addReactionToMessage(token, messageId, selectedEmoji)
-			// We replace the reaction details in the reactions store and wipe the old
-			// values
-			context.dispatch('updateReactions', {
+
+			const actorId = context.getters.getActorId()
+			const actorObject = response.data.ocs.data[selectedEmoji].find(actor => actor.actorId === actorId)
```

Comment:
We receive the same list of actual detailed reactions as a response from:
- `getReactionsDetails`
- `addReactionToMessage`
- `removeReactionFromMessage`

Why don't use `reactionsStore.updateReactions` with a full list instead of patching only one? optimization could be done in the store itself
Plus, someone else could react at the same time, and we will know it from the response (but you're omitting it now)